modules/authentication/package.json

```diff
@@ -9,9 +9,10 @@
   "scripts": {
     "start": "node dist/index.js",
     "prepare": "npm run build",
-    "build": "rimraf dist && tsc",
+    "build": "npm run generateTypes && rimraf dist && tsc",
```

Comment:
remove generateTypes command and the build inclusion and just add a prebuild command